multimedia/M2VDownsizer/Files/Makefile

```diff
@@ -0,0 +1,257 @@
+# Makefile for M2VDowsizer
+#
+# Author:   Robert Kennedy
+# EMAIL:    amtor@hotmail.com
+#
+# License for Makefile:  GPL-2
+#
+# 'make depend' uses makedepend to automatically generate dependencies 
+#               (dependencies are added to end of Makefile)
+# 'make'        build executable file 'M2VDownsizer'
+# 'make clean'  removes all .o and executable files
+#
+#  Note:    There is NO "make install"
+#           After running "make", just copy M2VDownsizer into the desired directory
+#           e.g.    sudo chmod +x ./M2VDownsizer
+#                   sudo cp M2VDownsizer /usr/local/bin
+#               
+
+# Define the C compiler to use
+#
+CC := clang
+
+# Define any compile-time flags
+#
+CFLAGS += -Wall -g
+#CFLAGS += -Wall -g -Wno-unused
+#CFLAGS += -g -w
+#CFLAGS ?= -mmacosx-version-min=10.7 -isysroot /Developer/SDKs/MacOSX10.7.sdk
+
+# Append additional architecture dependent compile-time flags
+#
+CFLAGS += -Xarch_ppc -faltivec
+CFLAGS += -Xarch_ppc64 -faltivec
+
+# Alternative code for appending additional architecture dependent compile-time flags
+# Note: Use as last resort
+#       It is better to use "-Xarch_ppc" etc as these CFLAGS will work 
+#       when cross-compiling
+#
+#UNAME_P := $(shell uname -p)
+
+#Multi-line comment!  Remove first ifeq and corresponding endif to uncomment
+ifeq ("x","y")
+ifeq ($(findstring ppc, $(UNAME_P)), ppc)
+	CFLAGS += -faltivec
+	CXXFLAGS += -flativec
+endif
+endif
+
+# Define any additional linker flags
+#
+LDFLAGS += 
+
+# Define any directories containing header files other than directory containing 
+# the system header files, typically /usr/include
+#   Statement Format: INCLUDES := -Iinclude_path
+#   E.g. INCLUDES := -I./  -I/opt/local/include
+#
+INCLUDES := -I./ -Iutils -Iutils/altivec -Impeg2dec -Impeg2enc \
+-I/System/Library/Frameworks/CoreServices.framework/Headers 
```

Comment:
You should never need to write an include flag for framework headers like this. `-framework CoreServices` (already specified below) should take care of it, no?

Reply:
It is whether a question whether the compiler (e.g clang) can find the headers for the macOS framework headers without them being explicitly specified.   On my Mac, I could not get the code to compile without specifying the location of the macOS framework headers in INCLUDES in the Makefile, if my memory serves me right.  (Please note that you do not need to specify the location of these headers when using XCode.  You just need to tell XCode that you want your code to link against a certain framework.  But that does not appear to be the case when you are not using XCode).   

INHO,  Apple wants developers to use XCode when using their frameworks.  

If you would like me to do some more testing, it will need to wait a couple of days util I get back home.

Reply:
A zillion ports are able to link with macOS frameworks just fine without using Xcode and I've never seen one use an absolute path to the framework headers like that.

Reply:
This has been resolved in my last push.  I removed `-I/System/Library/Frameworks/CoreServices.framework/Headers` from `INCLUDES`.  It is not needed to build the project.   And this deletion will help make the port more resilient to any unexpected changes to the location of these headers by Apple.

(But please note that this include is helpful for the `make depend` target to help properly build the list of header dependencies.  But these header dependencies are not needed to build the project via MacPorts).